apps/aurora-portal/src/client/Identity/Auth/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react"
+import type { AuroraReactQueryRouter } from "../../../../polaris-bff/routers"
+// @ts-expect-error missing types
+import { ButtonRow, Button, Form, FormRow, TextInput } from "@cloudoperators/juno-ui-components"
+import { useLocation } from "wouter"
+
+export function SignIn(props: { api: AuroraReactQueryRouter["identity"] }) {
+  const [form, setForm] = useState({ domain: "", user: "", password: "" })
+  const { api } = props
+  const tokenMutation = api.login.useMutation()
+  const setLocation = useLocation()[1]
+  const checkLoginStatus = api.getAuthStatus.useQuery()
+
+  const login = () => {
```

Comment:
shouldn't we use anonymous functions?

Reply:
yes, for components not for internal functions. I mean you can if you want to.